kyaml/setters2/set.go

```diff
@@ -200,15 +205,23 @@ func validateAgainstSchema(ext *cliExtension, sch *spec.Schema) error {
 	sc.Properties[ext.Setter.Name] = *sch
 
 	input := map[string]interface{}{}
-	format := `{"%s" : "%s"}`
-	if yaml.IsValueNonString(ext.Setter.Value) {
-		format = `{"%s" : %s}`
+	inputJSON := ""
```

Comment:
nit: I think it would be more idiomatic go to define these with `var inputJSON string`

Reply:
Done.